ark/type/__tests__/submodule.test.ts

```diff
@@ -29,7 +29,7 @@ contextualize.each(
 				Module<{
 					a: string
 					b: number
-					sub: Module<{
+					sub: Submodule<{
```

Comment:
Pretty confident this test update is necessary. It looks like the expected type was just written wrong. Like in all cases this update matches what's on `main`.